packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.tsx

```diff
@@ -74,7 +74,7 @@ function QueryWrapperInternal(props: QueryWrapperProps) {
     fileVersionColumnName,
   } = props
 
-  const hasSelectedRows = useAtomValue(hasSelectedRowsAtom)
+  const hasSelectedRows = useHasSelectedRowsAtomValue()
```

Comment:
ditto for simplfying mocking -- instead of trying to mock `useAtomValue` or faking `hasSelectedRowsAtom`, we can just mock `useHasSelectedRowsAtomValue`